Orange/widgets/data/owsave.py

```diff
@@ -175,14 +180,18 @@ def save_file(self):
                     os.path.join(
                         self.last_dir,
                         self.basename + self.type_ext + self.compress_ext),
-                    self.data)
+                    self.data, self.with_annotations,
+                    )
             except Exception as err_value:
                 self.error(str(err_value))
             else:
                 self.error()
 
     def update_extension(self):
         self.type_ext = [ext for name, ext, _ in FILE_TYPES if name == self.filetype][0]
+        self.annotations_cb.setEnabled(True)
+        if self.type_ext in ['.pkl', '.xlsx']:
```

Comment:
Whether this checkbox is shown should be decided by the file writer and not hard coded in the widget. See the general comment.